ios/NativeModules/TextileIPFS.m

```diff
@@ -96,8 +96,14 @@ - (dispatch_queue_t)methodQueue {
   NSData *data = [hashesString dataUsingEncoding:NSUTF8StringEncoding];
   id json = [NSJSONSerialization JSONObjectWithData:data options:0 error:nil];
   NSArray *hashes = [json objectForKey:@"hashes"];
+  NSArray *paths = [json objectForKey:@"paths"];
+  // TODO: Could we just return the json directly?
   if (hashes) {
-    resolve(hashes);
+    if (paths) {
+      resolve(@{ @"hashes": hashes, @"paths": paths });
+    } else {
+      resolve(@{ @"hashes": hashes });
```

Comment:
This should never happen and wouldn't make any sense if it did. Related to my comment about the data structure being returned having separate hashes and paths properties, when those things are actually related and should be grouped into an object.

Reply:
I’ll fix, don’t merge.